diff --git a/node_modules/@sentry/nextjs/dist/index.server.js b/node_modules/@sentry/nextjs/dist/index.server.js
index 4518433..9ac8646 100644
--- a/node_modules/@sentry/nextjs/dist/index.server.js
+++ b/node_modules/@sentry/nextjs/dist/index.server.js
@@ -3,7 +3,6 @@ var tslib_1 = require("tslib");
 var integrations_1 = require("@sentry/integrations");
 var node_1 = require("@sentry/node");
 var utils_1 = require("@sentry/utils");
-var instrumentServer_1 = require("./utils/instrumentServer");
 var metadataBuilder_1 = require("./utils/metadataBuilder");
 var userIntegrations_1 = require("./utils/userIntegrations");
 tslib_1.__exportStar(require("@sentry/node"), exports);
@@ -67,5 +66,4 @@ exports.withSentryConfig = config_1.withSentryConfig;
 var withSentry_1 = require("./utils/withSentry");
 exports.withSentry = withSentry_1.withSentry;
 // wrap various server methods to enable error monitoring and tracing
-instrumentServer_1.instrumentServer();
 //# sourceMappingURL=index.server.js.map
\ No newline at end of file
diff --git a/node_modules/@sentry/nextjs/esm/index.server.js b/node_modules/@sentry/nextjs/esm/index.server.js
index 5b10206..9b31ba6 100644
--- a/node_modules/@sentry/nextjs/esm/index.server.js
+++ b/node_modules/@sentry/nextjs/esm/index.server.js
@@ -1,7 +1,6 @@
 import { RewriteFrames } from '@sentry/integrations';
 import { configureScope, getCurrentHub, init as nodeInit, Integrations } from '@sentry/node';
 import { logger } from '@sentry/utils';
-import { instrumentServer } from './utils/instrumentServer';
 import { MetadataBuilder } from './utils/metadataBuilder';
 import { addIntegration } from './utils/userIntegrations';
 export * from '@sentry/node';
@@ -60,5 +59,4 @@ function addServerIntegrations(options) {
 export { withSentryConfig } from './config';
 export { withSentry } from './utils/withSentry';
 // wrap various server methods to enable error monitoring and tracing
-instrumentServer();
 //# sourceMappingURL=index.server.js.map
\ No newline at end of file
diff --git a/node_modules/@sentry/nextjs/src/index.server.ts b/node_modules/@sentry/nextjs/src/index.server.ts
index e8c096b..60b9f00 100644
--- a/node_modules/@sentry/nextjs/src/index.server.ts
+++ b/node_modules/@sentry/nextjs/src/index.server.ts
@@ -1,17 +1,18 @@
-import { RewriteFrames } from '@sentry/integrations';
-import { configureScope, getCurrentHub, init as nodeInit, Integrations } from '@sentry/node';
-import { logger } from '@sentry/utils';
+import {RewriteFrames} from '@sentry/integrations';
+import {configureScope, getCurrentHub, init as nodeInit, Integrations} from '@sentry/node';
+import {logger} from '@sentry/utils';
+import {MetadataBuilder} from './utils/metadataBuilder';
+import {NextjsOptions} from './utils/nextjsOptions';
+import {addIntegration} from './utils/userIntegrations';
 
-import { instrumentServer } from './utils/instrumentServer';
-import { MetadataBuilder } from './utils/metadataBuilder';
-import { NextjsOptions } from './utils/nextjsOptions';
-import { addIntegration } from './utils/userIntegrations';
 
 export * from '@sentry/node';
-
 // Here we want to make sure to only include what doesn't have browser specifics
 // because or SSR of next.js we can only use this.
-export { ErrorBoundary, withErrorBoundary } from '@sentry/react';
+export {ErrorBoundary, withErrorBoundary} from '@sentry/react';
+export {withSentryConfig} from './config';
+export {withSentry} from './utils/withSentry';
+
 
 /** Inits the Sentry NextJS SDK on node. */
 export function init(options: NextjsOptions): void {
@@ -72,8 +73,5 @@ function addServerIntegrations(options: NextjsOptions): void {
   }
 }
 
-export { withSentryConfig } from './config';
-export { withSentry } from './utils/withSentry';
 
 // wrap various server methods to enable error monitoring and tracing
-instrumentServer();
